src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,83 @@
+import cn from 'classnames';
+import { useState, useMemo, useRef } from 'react';
+import { Person } from '../types/Person';
+
+type Props = {
+  onSelected: (person: Person) => void;
+  people: Person[]
+};
+
+export const Autocomplete: React.FC<Props> = ({ onSelected, people }) => {
+  const [query, setQuery] = useState('');
+  const [focusInput, setFocusInput] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [IsFocusInput, setIsFocusInput] = useState(false);
```